src/sonic-yang-models/ietf-yang-types.yang

```diff
@@ -0,0 +1,474 @@
+module ietf-yang-types {
```

Comment:
Same, I think this file should not be here.

Reply:
Same as above comment

Reply:
Removed the ietf yang models